tests/test_events/test_api.py

```diff
@@ -223,6 +239,48 @@ def test_clear(self) -> None:
         assert self.event_api.sent_events == []
         assert self.event_api.pending_events == []
 
+    @patch.object(Boto3Client, "eventbridge", MagicMock())
+    def test_clear_pending_clears_only_pending_events(self) -> None:
```

Comment:
Please, stay consistent with the naming convention that we agreed to follow in all repositories.